city_scrapers/spiders/il_investment_policy.py

```diff
@@ -0,0 +1,157 @@
+import re
+from collections import defaultdict
+from datetime import date, datetime, time
+from enum import Enum
+
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlInvestmentPolicySpider(CityScrapersSpider):
+    name = "il_investment_policy"
+    agency = "Illinois Investment Policy Board"
+    timezone = "America/Chicago"
+    allowed_domains = ["www2.illinois.gov"]
+    start_urls = ["https://www2.illinois.gov/sites/iipb/Pages/MeetingInformation.aspx"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        self._validate_location(response)
+        links = self._parse_links(response)
+
+        for item in response.xpath(
```

Comment:
It looks like this is missing the upcoming meeting in December, and we'll want to get that as well. One way of handling this would be to get all of the meeting dates from the keys in the `links` `defaultdict`, add any dates found in the upcoming meetings section, and then iterate through that similar to this:

https://github.com/City-Bureau/city-scrapers-cle/blob/a499b5cb2ca28e73a73823b3e67d478f2b87da0c/city_scrapers/spiders/cuya_planning.py#L31-L40

Let me know if that makes sense!